iree/tools/iree-e2e-matmul-test.c

```diff
@@ -693,7 +745,7 @@ int main(int argc, char** argv) {
   if (!iree_status_is_ok(status)) {
     iree_status_fprint(stderr, status);
     iree_status_free(status);
-    return 1;
+    return iree_status_is_cancelled(status) ? EXIT_SUCCESS : EXIT_FAILURE;
```

Comment:
This is shady - it'll work, but only by chance due to the way status codes are encoded in the status pointer that you free'd above. You can move it above as an `int ret = is_cancelled... ? EXIT_STATUS ....` before you free.

Reply:
good catch, thanks!